src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/Country.java

```diff
@@ -227,7 +227,7 @@ public enum Country {
     SWEDEN("Sweden"),
     SWITZERLAND("Switzerland"),
     SYRIA("Syria"),
-    TAIWAN_PROVINCE_OF_CHINA("Taiwan, Province of China"),
+    TAIWAN("Taiwan"),
```

Comment:
The spec (schema.json file) also needs updating to reflect this change in name.

Reply:
I leave this for another PR for ROE-1254 and back the Taiwan change out.